Automattic/woocommerce-payments
# Fix: Javascript not loading on order screens with HPOS enabled

Fixes #6967 

#### Changes proposed in this Pull Request

Order screen javascript is now enqueued on admin order pages when HPOS is enabled.

With HPOS, WooCommerce introduces a different way of rendering admin order screens. With CPT orders the screen id is `shop_order`. With HPOS enabled the screen id is `woocommerce_page_wc-orders`. This prevented the expected javascript from being loaded on page.

#### Testing instructions

1. Enable HPOS.
2. Click on a WooPayments order
3. Click `Refund`
4. The `Refund Manually` button should not be visible (this is removed with the javascript in `client/order/index.js`)

![Screenshot 2023-08-10 at 12 23 41 PM](https://github.com/Automattic/woocommerce-payments/assets/57298/ab233091-fa5e-40f5-b084-005a859fc62d)

Additionally, multicurrency settings should now be visible on the order page with HPOS enabled.

```
<script>
   woocommerce_admin_meta_boxes.currency_format_num_decimals = 2;
   woocommerce_admin_meta_boxes.currency_format_decimal_sep = ',';
   woocommerce_admin_meta_boxes.currency_format_thousand_sep = '.';
   woocommerce_admin_meta_boxes.currency_format = '%v %s';
   woocommerce_admin_meta_boxes.rounding_precision = 2;
</script>
```


-------------------


**Post merge**
